include/Core/UI/Setting/CoreSetting.php

```diff
@@ -281,8 +281,10 @@ public function render_db_size_notification_settings( $args ) {
 					'<b>Save</b>'
 				); ?>
             </p>
-		<?php endif; ?>
 		<?php
+		endif;
+
+		do_action( 'el_after_db_size_notification_setting' );
```

Comment:
I have renamed the hook. Generally we should name the hook based on the place where it is invoked and not how the function that will use the hook.

Add the phpdoc block for this hook.

Reply:
Fixed.